packages/trader/src/AppV2/Utils/helper.ts

```diff
@@ -0,0 +1,26 @@
+import { formatDuration, getDiffDuration } from '@deriv/shared';
+
+type GetCardLabels = () => {
+    DAY: string;
+    DAYS: string;
+};
+
+type TRemainingTimeProps = {
+    end_time?: number;
+    start_time: moment.Moment;
+    format?: string;
+    getCardLabels: GetCardLabels;
+};
+
+export const getRemainingTime = ({ end_time, format, getCardLabels, start_time }: TRemainingTimeProps) => {
```

Comment:
this is not being used. should remove and its tests

Reply:
Removed